package.json

```diff
@@ -643,18 +644,20 @@
     ]
   },
   "scripts": {
-    "esbuild-base": "esbuild ./src/extension.ts --bundle --outfile=out/extension.js --external:vscode --format=cjs --platform=node",
-    "esbuild": "npm run esbuild-base -- --sourcemap",
-    "esbuild-watch": "npm run esbuild-base -- --sourcemap --watch",
-    "compile": "npm run esbuild",
-    "watch": "npm run download:artifacts && npm run esbuild-watch",
+    "prepare": "npm run download:artifacts",
+    "compile": "webpack",
+    "compile:test": "tsc -p ./",
+    "compile:prod": "webpack --mode production --devtool hidden-source-map",
+    "watch": "webpack --watch",
+    "web": "npm run compile && vscode-test-web --browserType=chromium --extensionDevelopmentPath=. ./testFixture",
+    "web:serve": "npx serve --cors -l 5000",
+    "web:tunnel": "npx localtunnel -p 5000",
```

Comment:
I ran into a problem locally on macOS due to the Control Center already listening on port 5000 (which apparently is configured like that by default on a fresh macOS installation):
https://developer.apple.com/forums/thread/682332

Is it worth picking a higher port number, maybe just `5001`, to avoid the conflict?

Reply:
Changed!